crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -22,17 +22,15 @@ use diesel::{
     ExpressionMethods, PgConnection, QueryDsl, RunQueryDsl,
 };
 use serde_json::{json, Map, Value};
-use service_utils::{
-    bad_argument, db_error, result as superposition,
-    service::types::{AppHeader, DbConnection},
-    unexpected_error,
-};
+use superposition_macros::{bad_argument, db_error, unexpected_error};
+use superposition_types::{result as superposition, result::AppError, User};
```

Comment:
This might be out of the scope of PR, but can you remove import of `result::AppError` as it is already being included when we import `result` as `superposition`, and also make the necessary changes in the code.